style.css

```diff
@@ -0,0 +1,83 @@
+:root {
+  --quiz-bg: #dfe2eb;
+  --quiz-bg-dark: #272a2c;
+  --quiz-header: #0e131f;
+  /* --primary-color: rgb(204, 31, 31); */
+  --wrong-option: red;
+  --right-option: green;
+}
+i {
+  margin-right: 0.5rem;
+}
+.grid-3 {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  margin: 3rem 0;
+}
+.quiz-body {
+  background-color: var(--quiz-bg);
+  color: var(--primary-text);
+  height: 100vh;
+  display: grid;
+  grid-template-rows: auto 1fr auto;
+  grid-template-columns: 1fr 1fr;
+  grid-template-areas:
+    "header header"
+    "main   main"
+    "footer footer";
+}
+.quiz-header {
+  grid-area: header;
+  padding: 3rem;
+  background: var(--quiz-header);
+  justify-content: space-between;
+  color: var(--light-text);
+}
+.logo,
+.right-side > * {
+  padding: 1rem;
+  cursor: pointer;
+}
+.right-side {
+  gap: 4rem;
+}
+.quiz-main {
+  grid-area: main;
+  overflow: scroll;
+}
+main::--webkit-scrollbar {
+  width: 0.5rem;
+  height: 1rem;
+}
+main::--webkit-scrollbar-thumb {
+  background: var(--primary-color);
+}
+.quiz-footer {
```

Comment:
Add the comment for the webkit scrollbar css, so that the reviewer could understand that it is for hiding the scrollbar

Reply:
Okay will add